variables.tf

```diff
@@ -15,6 +15,16 @@ variable "min_capacity" {
   default     = 2
 }
 
+variable "send_access_logs_to_log_analytics" {
+  description = "Send access logs to log analytics workspace, this can be quite expensive on busy FrontDoor instances so disable it and send to Storage account instead"
```

Comment:
```suggestion
  description = "Send access logs to log analytics workspace, this can be quite expensive on busy instances so disable it and send to Storage account instead"
```